### DISCUSSION TITLE
Role for Bootstrapping

### DISCUSSION THREAD

**Discussion by chr78** [uyuni-project/uyuni#4783] [Ideas]
Hi there,

t seems Bootstrapping is only possible for the role "organization admins".

It would be great to delegate this right to a special role.
We are a small team and everybody has their own machines and it makes no sense that everybody will become an Organization Admin just to register their machines.

**Comment by maceoparkore** [comment]
the bootstrapping is available to everyone, accept keys is indeed only available by org admin, you can configure autoaccept keys with restricted grains.

I'm agree with you it will be usefull to delegate this functionnality to a dedicated role and not the ORG Admin, the reactor behind acceptance has to create entries both to salt master and postgres db isuspect only the org admin to have the privileges to do that.
regards
